src/AppInstallerCommonCore/Public/winget/ManifestCommon.h

```diff
@@ -124,6 +124,11 @@ namespace AppInstaller::Manifest
         std::vector<string_t> WindowsLibraries;
         std::vector<PackageDependency> PackageDependencies;
         std::vector<string_t> ExternalDependencies;
+
+        bool HasAny() const {

```

Comment:
Regarding moving to a unified dependency model, I would suggest having the manifest simply have a `std::vector<Dependency>` as well.